client/jetpack-connect/authorize.js

```diff
@@ -101,6 +101,8 @@ export class JetpackAuthorize extends Component {
 		const { recordTracksEvent } = this.props;
 
 		const tracksProperties = pick( this.props.authQuery, 'from' );
+		tracksProperties[ 'is-mobile-app-flow' ] = this.props.isMobileAppFlow;
```

Comment:
Use `_` not `-`

> Property names should also be lowercase and use underscores instead of dashes

PCYsg-4S2-p2

Reply:
With that change, we can probably use `.` access over `[]` as well. The linter will likely force that.

Reply:
> Use _ not -

Ah thanks! I always get this wrong.